[2022-07-18 23:47:17,626] {taskinstance.py:1160} INFO - Dependencies all met for <TaskInstance: local_to_gcs.local_to_gcs_task scheduled__2005-01-10T06:00:00+00:00 [queued]>
[2022-07-18 23:47:17,638] {taskinstance.py:1160} INFO - Dependencies all met for <TaskInstance: local_to_gcs.local_to_gcs_task scheduled__2005-01-10T06:00:00+00:00 [queued]>
[2022-07-18 23:47:17,639] {taskinstance.py:1357} INFO - 
--------------------------------------------------------------------------------
[2022-07-18 23:47:17,640] {taskinstance.py:1358} INFO - Starting attempt 2 of 2
[2022-07-18 23:47:17,641] {taskinstance.py:1359} INFO - 
--------------------------------------------------------------------------------
[2022-07-18 23:47:17,654] {taskinstance.py:1378} INFO - Executing <Task(PythonOperator): local_to_gcs_task> on 2005-01-10 06:00:00+00:00
[2022-07-18 23:47:17,668] {standard_task_runner.py:52} INFO - Started process 40198 to run task
[2022-07-18 23:47:17,673] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'local_to_gcs', 'local_to_gcs_task', 'scheduled__2005-01-10T06:00:00+00:00', '--job-id', '77', '--raw', '--subdir', 'DAGS_FOLDER/local_to_gcs.py', '--cfg-path', '/tmp/tmpnceljg7d', '--error-file', '/tmp/tmpflcvrogw']
[2022-07-18 23:47:17,675] {standard_task_runner.py:80} INFO - Job 77: Subtask local_to_gcs_task
[2022-07-18 23:47:17,798] {task_command.py:370} INFO - Running <TaskInstance: local_to_gcs.local_to_gcs_task scheduled__2005-01-10T06:00:00+00:00 [running]> on host 32d1f26135d5
[2022-07-18 23:47:17,899] {taskinstance.py:1572} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=local_to_gcs
AIRFLOW_CTX_TASK_ID=local_to_gcs_task
AIRFLOW_CTX_EXECUTION_DATE=2005-01-10T06:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=2
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2005-01-10T06:00:00+00:00
[2022-07-18 23:47:26,067] {python.py:173} INFO - Done. Returned value was: None
[2022-07-18 23:47:26,142] {taskinstance.py:1401} INFO - Marking task as SUCCESS. dag_id=local_to_gcs, task_id=local_to_gcs_task, execution_date=20050110T060000, start_date=20220718T234717, end_date=20220718T234726
[2022-07-18 23:47:26,266] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-07-18 23:47:26,328] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
